core/storage/collection/gae_models.py

```diff
@@ -88,6 +108,7 @@ def get_export_policy(cls):
 
     @classmethod
     def get_instance_id(cls, collection_id, version):
+        # type: (Text, Union[int, Text]) -> Text
```

Comment:
Why is the second arg a "Union[int, Text]"? It should be one or the other.

Reply:
We have been passing both `int` and `str` values for version in our codebase knowingly or unknowingly.

Reply:
Then let's fix that and standardize it.

Reply:
Will do

Reply:
Let's keep it to `str`?

Reply:
Int seems more sensible, since versions are ints. Why str?

Reply:
I'll move forward with int. Thanks!

Reply:
Done

Reply:
Have you checked that all callers of this method pass in int?

Reply:
Yes for this method everyone references using int.